recipients_app/lib/view/widgets/income/balance_card/balance_card_grid.dart

```diff
@@ -0,0 +1,48 @@
+import "package:app/data/models/payment/mapped_payment.dart";
+import "package:app/data/models/payment/payment_ui_status.dart";
+import "package:app/view/widgets/income/payment_status_icon.dart";
+import "package:flutter/material.dart";
+
+const kMaxMonths = 36;
```

Comment:
I extracted those consts to make it more understandable. First I was like "why 36?" and then I remembered. (we can decide if we want to have the k before the name or not). it's darts default for consts.